pocs/mount/mount.py

```diff
@@ -514,28 +511,38 @@ def slew_to_target(self):
 
         return success
 
-    def slew_to_home(self):
+    def slew_to_home(self, blocking=False):
```

Comment:
Should this have a maximum time to block (e.g. 10 minutes)?

Reply:
Yes, agreed. I almost marked this WIP until I could add that in as it does seem like a potential bad failure. I'll add before merge and re-request a review. Thanks!

Reply:
I've added a timeout, however it's not clear to me what the behavior should be. We slew to home before we park, so if we have a timeout while slewing to home we can't actually then attempt a park.  Note also that #889 potential adds some confusion to this as that new park behaviour is based off movement from the home position.

> FWIW, I'm reminded by reading the description of this PR that I encountered a problem where slewing never ended, so the dome was stuck open until I noticed a day+ later.

IIRC you were stuck in a slew to target but not a slew to home. We would need to add a timeout there as well.